### ISSUE TITLE
PROMETHEUS_SCRAPE_PORT environment variable not working

### ISSUE THREAD

**Issue by asviel** [thumbor-community/prometheus#5]
Hi,

I'm trying to use PROMETHEUS_SCRAPE_PORT environment variable to configure my instance, but I get the following error:

```
Traceback (most recent call last):
  File "/app/bin/thumbor", line 8, in <module>
    sys.exit(main())
  File "/app/lib/python3.8/site-packages/thumbor/server.py", line 150, in main
    with get_context(server_parameters, config, importer) as context:
  File "/app/lib/python3.8/site-packages/thumbor/server.py", line 101, in get_context
    return Context(server=server_parameters, config=config, importer=importer)
  File "/app/lib/python3.8/site-packages/thumbor/context.py", line 41, in __init__
    self.metrics = importer.metrics(config)
  File "/app/lib/python3.8/site-packages/tc_prometheus/metrics/prometheus_metrics.py", line 19, in __init__
    start_http_server(config.PROMETHEUS_SCRAPE_PORT)
  File "/app/lib/python3.8/site-packages/prometheus_client/exposition.py", line 170, in start_wsgi_server
    httpd = make_server(addr, port, app, TmpServer, handler_class=_SilentHandler)
  File "/usr/local/lib/python3.8/wsgiref/simple_server.py", line 154, in make_server
    server = server_class((host, port), handler_class)
  File "/usr/local/lib/python3.8/socketserver.py", line 452, in __init__
    self.server_bind()
  File "/usr/local/lib/python3.8/wsgiref/simple_server.py", line 50, in server_bind
    HTTPServer.server_bind(self)
  File "/usr/local/lib/python3.8/http/server.py", line 138, in server_bind
    socketserver.TCPServer.server_bind(self)
  File "/usr/local/lib/python3.8/socketserver.py", line 466, in server_bind
    self.socket.bind(self.server_address)
TypeError: an integer is required (got type str)
```

This is because the value of the environment variable is a string, while the function expects integer
https://github.com/prometheus/client_python/blob/master/prometheus_client/exposition.py#L162

thumbor 7.0.9
prometheus-client 0.14.1

Can you fix this?

**Comment by maximka777** [comment]
I'll address it in https://github.com/thumbor-community/prometheus/pull/6

**Comment by savar** [comment]
thanks @mbarouski .. I released version 1.0.0 of the plugin and from my local tests this works now.. I will close this issue but @asviel please re-open if you still experience issues with it

be aware that the plugin works with python3/thumbor7 now. I am not sure if it still works with older versions.
